content/en/tracing/trace_collection/custom_instrumentation/php.md

```diff
@@ -417,6 +417,44 @@ function doRiskyThing() {
 {{% /tab %}}
 {{< /tabs >}}
 
+## Adding span links (Beta)
+
+<div class="alert alert-info">Support for span links is in beta and requires the <a href="https://github.com/DataDog/dd-trace-php/releases/tag/0.87.2">PHP tracer v0.87.2+</a></div>
+
+Span links associate one or more spans together that don't have a typical parent-child relationship. They may associate spans within the same trace or spans across different traces.
+
+Span links help trace operations in distributed systems, where workflows often deviate from linear execution patterns. Additionally, span links are useful to trace the flow of operations in systems that execute requests in batches or process events asynchronously.
+
+To add a span link from an existing span:
+
+```php
+$spanA = \DDTrace\start_trace_span();
+$spanA->name = 'spanA';
+$distributedTracingHeaders = \DDTrace\generate_distributed_tracing_headers();
+\DDTrace\close_span();
+
+$spanB = \DDTrace\start_trace_span();
+$spanB->name = 'spanB';
+$spanB->links[] = \DDTrace\SpanLink::fromHeaders($distributedTracingHeaders);
+\DDTrace\close_span();
+```
+
+To link a span using distributed tracing headers:
+
+```php
+$spanA = \DDTrace\start_trace_span();
+$spanA->name = 'spanA';
+$distributedTracingHeaders = \DDTrace\generate_distributed_tracing_headers();
+\DDTrace\close_span();
+
+$spanB = \DDTrace\start_trace_span();
+$spanB->name = 'spanB';
+$spanB->links[] = $spanA->getLink();
+\DDTrace\close_span();
+```
```

Comment:
I think this is the code snippet for "To add a span link from an existing span:", and vice-versa :smiley: